src/configuration.ts

```diff
@@ -63,9 +60,9 @@ export class Configuration {
 		}
 		const baseUri = this.getState('baseUri');
 
-		const debugChannel = this.ctx.workspaceState.get('debugChannel', '');
-		const debugCipher = this.ctx.workspaceState.get('debugCipher', '');
```

Comment:
I found references to debugChannel and debugCipher in a different file [ConfigurationMenu](https://github.com/launchdarkly/ld-vscode/blob/058c390aa47332b49c6efeda1123b33eb8ade0d2/src/configurationMenu.ts#L25). These seem to be related so maybe need to be removed as well?

Reply:
@yusinto yep, just pushed a PR cleaning these up also.